runtime/doc/lsp.txt

```diff
@@ -158,6 +158,27 @@ name: >
   vim.lsp.protocol.TextDocumentSyncKind.Full == 1
   vim.lsp.protocol.TextDocumentSyncKind[1] == "Full"
 
+================================================================================
+LSP HIGHLIGHT GROUPS                                      *lsp-highlight-groups*
+
+                                                        *hl-LspDiagnosticsError*
+LspDiagnosticsError       used for 'Error' diagnostic virtual text
```

Comment:
do not use single quotes in vim help docs, unless referring to options.
```suggestion
LspDiagnosticsError       used for "Error" diagnostic virtual text
```

Reply:
Fixed remaining issues